docs/current_docs/partials/_install-cli.mdx

```diff
@@ -30,7 +30,7 @@ If you do not have Homebrew installed, or you want to install a specific version
 
 ```shell
 cd /usr/local
-curl -fsSL https://dl.dagger.io/dagger/install.sh | DAGGER_VERSION=0.13.3 sh
+curl -fsSL https://dl.dagger.io/dagger/install.sh | DAGGER_VERSION=0.13.3 sudo -E sh
```

Comment:
The command below does the exact same thing so this is a bit redundant. Maybe here we can just remove the `cd /usr/local` instead of adding the `-E`? This way the binary will be downloaded into `$workdir/bin/dagger` which is useful for some people that don't like adding `sudo` to their curl | bash installs.